src/hearings/store/effects/hearing-request.effects.ts

```diff
@@ -61,32 +62,52 @@ export class HearingRequestEffects {
   public continueNavigation$ = this.actions$.pipe(
     ofType(hearingRequestActions.UPDATE_HEARING_REQUEST),
     tap(() => {
-      const nextPage = this.pageFlow.getNextPage(this.screenNavigations$);
+      let nextPage;
       switch (this.mode) {
         case Mode.CREATE:
+          nextPage = this.pageFlow.getNextPage(this.screenNavigations$);
           if (nextPage) {
             return this.router.navigate(['hearings', 'request', nextPage]);
           } else {
             throw new Error('Next page not found');
           }
         case Mode.CREATE_EDIT:
+          nextPage = this.pageFlow.getNextPage(this.screenNavigations$);
           if (nextPage === 'hearing-welsh') {
```

Comment:
can we make use of enum instead of hard-coded string

Reply:
thanks this is good point I will make a change